packages/components-js/tests/vrt/playwright/specs/custom/banner.vrt.ts

```diff
@@ -9,7 +9,7 @@ test.describe(component, async () => {
 
   [1920, 2560, 3000].forEach((viewportWidth) => {
     test(`should have no visual regression for viewport ${viewportWidth}`, async ({ page }) => {
-      await setupScenario(page, `/${component}`, viewportWidth);
+      await setupScenario(page, `/${component}`, viewportWidth, { revertAutoFocus: true });
```

Comment:
Auto-Focus is a desired feature of this component but is now not tested via VRT anymore, is this by accident or a new requirement?

Reply:
It improves VRT reliability, especially for iFrame VRT pages but also others. I think we should test the auto focus behaviour by E2E instead.

Reply:
I've just added it locally now and removed `revertAutoFocus` option of `setupScenario()`. We can bring it back in a separate PR.